engine/lib/navigation.php

```diff
@@ -693,6 +693,7 @@ function _elgg_setup_vertical_menu(\Elgg\Hook $hook) {
 		$parent = $selected_item->getParent();
 		while ($parent instanceof \ElggMenuItem) {
 			$parent->setSelected();
+			$parent->addItemClass('elgg-menu-item-has-selected-child');
```

Comment:
Can is it just be `has-selected-child`?
Imagine targeting this with `.elgg-menu-item.elgg-menu-item-has-selected-child`

Reply:
will change it to `elgg-has-selected-child` OK?